crates/client/index.html

```diff
@@ -1,11 +0,0 @@
-<!DOCTYPE html>
```

Comment:
This doesn't seem to work without that file, why do you remove it?

Reply:
I will fix it.